digital_land_frontend/render.py

```diff
@@ -279,14 +288,16 @@ def format_name(name):
     return re_strip.sub(" ", name).title()
 
 
-def slug_to_breadcrumb(slug):
+def slug_to_breadcrumb(slug, reference=None):
     logging.debug(">> slug_to_breadcumb(%s)", slug)
     next_relative_path = "../"
     breadcrumb = []
     first_item = True
     for part in slug.split("/")[-1:0:-1]:
         if first_item:
             text = part
+            if reference:
```

Comment:
Slightly neater to make it ternary like you did above 
`text = reference if reference else part`